src/features/meta.ts

```diff
@@ -91,6 +91,7 @@ export default (api: IApi) => {
       tplPath: require.resolve('../templates/meta/runtime.ts.tpl'),
       context: {
         deepmerge: winPath(path.dirname(require.resolve('deepmerge/package'))),
+        rc_util: winPath(path.dirname(require.resolve('rc-util'))),
```

Comment:
需要改成这样否则路径是 `/path/to/node_modules/rc-util/lib` 而不是 `/path/to/node_modules/rc-util`：

```suggestion
        rc_util: winPath(path.dirname(require.resolve('rc-util/package'))),
```

Reply:
确实